src/commons/components/log-in-form/log-in-form.tsx

```diff
@@ -1,30 +1,54 @@
-import React, { Component } from 'react';
+import React from 'react';
+
+// @material
+import { Box, Button, Container, CssBaseline, InputBase, Typography } from '@material-ui/core';
 
 // styles
-import './log-in-form.css'
+import useStyles from './log-in-form-styles';
 
-interface Props {}
+interface Props { }
 
-interface State {}
+interface State { }
 
-export default class LogInForm extends Component<Props, State> {
-    render() {
-        return <article className="log-in-form">
-            <section >
-                <h3>Authorize</h3>
-                <form>
-                    <label htmlFor="user_email">E-mail:</label>
-                    <input name="email" type="text" id="user_email" />
-                    <p></p>
-                    
-                    <label htmlFor="user_password">Password:</label>
-                    <input name="password" type="password" id="user_password" />
-                    <p></p>
+const LogInForm: React.FC = () => {
```

Comment:
Why you create props and state interface and dont use from React.FC?
React.FC<Props, State>